### PR TITLE
Added XGBoost Classifier

### PR DESCRIPTION
**Pull request by mihir-yadav**
## Description
Used XGBoost Classifier to predict the likelihood of breast cancer, achieving an accuracy of over 96.5% on the test set.

## Affected Dependencies
xgboost, sklearn and matplotlib need to be installed.

## How has this been tested?
Trained and tested on sklearn's breast cancer dataset, achieving an accuracy of over 96.5%.

## Checklist
- [x] I have followed the [Contribution Guidelines](https://github.com/OpenMined/.github/blob/master/CONTRIBUTING.md) and [Code of Conduct](https://github.com/OpenMined/.github/blob/master/CODE_OF_CONDUCT.md)
- [x] I have commented my code following the [OpenMined Styleguide](https://github.com/OpenMined/.github/blob/master/STYLEGUIDE.md)
- [x] I have labeled this PR with the relevant [Type labels](https://github.com/OpenMined/.github/labels?q=Type%3A)
- [x] My changes are covered by tests

### CODE REVIEW

**Review by ramesht007 — APPROVED** [review]
LGTM!
